airflow/providers/apache/livy/hooks/livy.py

```diff
@@ -448,7 +448,7 @@ def _validate_extra_conf(conf: dict[Any, Any]) -> bool:
         if conf:
             if not isinstance(conf, dict):
                 raise ValueError("'conf' argument must be a dict")
-            if any(True for k, v in conf.items() if not (v and isinstance(v, str) or isinstance(v, int))):
+            if not all(v and isinstance(v, str) or isinstance(v, int) for v in conf.values()):
```

Comment:
```suggestion
            if not all(v and isinstance(v, (int, str)) for v in conf.values()):
```

Reply:
I already tried this at the beginning, but reverted it later. If `v = 0`, then:

```
>>> v and isinstance(v, str) or isinstance(v, int)
True

>>> v and isinstance(v, (str, int))
0
```

Reply:
Oh because operation precedence. Maybe add paranthese to make it more obvious?

```python
not (v and isinstance(v, str)) and not isinstance(v, int)
```

Reply:
Added them on all three occurrences.